bundles/org.openhab.binding.bluetooth.grundfosalpha/src/main/resources/OH-INF/i18n/grundfosalpha.properties

```diff
@@ -0,0 +1,23 @@
+# add-on
+
+addon.grundfosalpha.name = GrundfosAlpha Binding
+addon.grundfosalpha.description = This is the binding for Grundfos Alpha circulator pumps.
+
```

Comment:
This should probably be removed after the removal of the addon.xml file. If you then regenerate this file (see
https://www.openhab.org/docs/developer/utils/i18n.html#generating-i18n-properties-file), the changed text for `channel-type.grundfosalpha.grundfos-head.description` below should also be updated.